### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by RaphaelSve**
## Hey, I just made a Pull Request!

Add French translation to the Readme

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: README-fr_FR.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+[![Codecov](https://img.shields.io/codecov/c/github/backstage/backstage)](https://codecov.io/gh/backstage/backstage)
+[![](https://img.shields.io/github/v/release/backstage/backstage)](https://github.com/backstage/backstage/releases)
+[![Uffizzi](https://img.shields.io/endpoint?url=https%3A%2F%2Fapp.uffizzi.com%2Fapi%2Fv1%2Fpublic%2Fshields%2Fgithub.com%2Fbackstage%2Fbackstage)](https://app.uffizzi.com/ephemeral-environments/backstage/backstage)
+[![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7678/badge)](https://bestpractices.coreinfrastructure.org/projects/7678)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/backstage/backstage/badge)](https://securityscorecards.dev/viewer/?uri=github.com/backstage/backstage)
+
+## Qu'est-ce que Backstage?
+
+[Backstage](https://backstage.io/) est un framework open source conçu pour créer des portails à destination des développeur. Avec l'idée de fournir un catalogue d'applications centralisé, Backstage remet de l'ordre dans vos microservices et votre infrastructure, ce qui permet à vos équipes produit de coder rapidement et de manière efficace sans perdre en autonomie.
```
Comment (tcardonne) [inline README-fr_FR.md:19]:
```suggestion
[Backstage](https://backstage.io/) est un framework open source conçu pour créer des portails à destination des développeurs. Avec l'idée de fournir un catalogue d'applications centralisé, Backstage remet de l'ordre dans vos microservices et votre infrastructure, ce qui permet à vos équipes produit de coder rapidement et de manière efficace sans perdre en autonomie.
```

File: README-fr_FR.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Backstage rassemble tous vos outils d'infrastructure, services et documentation pour créer un environnement de développement fluide de bout en bout.
```
Comment (tcardonne) [inline README-fr_FR.md:21]:
```suggestion
Backstage rassemble tous vos outils d'infrastructure, services et documentations pour créer un environnement de développement fluide de bout en bout.
```

File: README-fr_FR.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+## Communauté
+
+Si vous voulez contribuer et vous impliquer dans notre communauté, voici les ressources à votre disposition :
+
+- [Discord chatroom](https://discord.gg/backstage-687207715902193673) - Pour obtenir de l'aide ou discuter du projet
+- [Contributing to Backstage](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md) - Rendez-vous ici si vous souhaitez contribuer
+- [RFCs](https://github.com/backstage/backstage/labels/rfc) - Contribuez à définir la direction technique
+- [FAQ](https://backstage.io/docs/FAQ)
+- [Code of Conduct](CODE_OF_CONDUCT.md) - C'est comme ça qu'on fonctionne
+- [Adopters](ADOPTERS.md) - Les entreprises utilisent déjà Backstage
```
Comment (tcardonne) [inline README-fr_FR.md:61]:
```suggestion
- [Adopters](ADOPTERS.md) - Les entreprises utilisant déjà Backstage
```

**Review by tcardonne — COMMENTED** [review]
Some small suggestions

**Comment by freben** [comment]
Hi! You'll note that `vale`, the spell checker, became upset. Could you find a way for vale to ignore that file entirely?

**Review by RaphaelSve — COMMENTED** [review]
Translation confirmed

**Comment by RaphaelSve** [comment]
@freben Maybe modify the IGNORED_WHEN_LISTING variable in the check-docs-quality.js file to include this regex :
 **/^README-[a-z]{2}_[A-Z]{2}\.md$/**

**Comment by freben** [comment]
feel free to do so as part of this PR!

**Review by awanlin — APPROVED** [review]
Merci @RaphaelSve 🚀

**Comment by awanlin** [comment]
@freben, this looks good to me, just need another approval 👍

**Review by freben — APPROVED** [review]
Ooh très bien!

**Pull request merged**